### PR TITLE
FixL: testing elmer envision 

### PR DESCRIPTION
**Pull request by mwkyuen**
Plus some docstring. `example01_missing_header.csv` is missing `Plate information` header on L21

### CODE REVIEW

**Review by mdlu — APPROVED** [review]

File: tests/parsers/perkin_elmer_envision/testdata/example01_missing_header.csv [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+A,31441,31278,31727,,32350,32415,32847,,,,,,
+B,19603,19063,19621,,19543,19799,20087,,,,,,
+C,12762,12502,12582,,12842,13071,12914,,,,,,
+D,8153,8018,8093,,8387,8142,8294,,,,,,
+E,5792,5630,5882,,6063,5956,6096,,,,,,
+F,4671,4477,4639,,4854,4728,4822,,,,,,
+G,3887,3814,3954,,3979,4044,4139,,,,,,
+H,3266,3354,3252,,3405,3342,3339,,,,,,
+
+Plate information
```
Comment (nathan-stender) [inline tests/parsers/perkin_elmer_envision/testdata/example01_missing_header.csv:40]: Can we drop the rest of this file, in the interest of keeping this testing data as succinct as possible

**Review by nathan-stender — CHANGES_REQUESTED** [review]

File: tests/parsers/perkin_elmer_envision/testdata/example01_missing_header.csv (hunk 1 above)
Comment (mwkyuen) [reply tests/parsers/perkin_elmer_envision/testdata/example01_missing_header.csv]: Fixed in [d71594e](https://github.com/Benchling-Open-Source/allotropy/pull/72/commits/d71594e2935c91fd5bb0383205a1e1ad5953c605)

File: tests/parsers/perkin_elmer_envision/elmer_envision_parser_test.py [hunk 2]
```diff
@@ -37,3 +38,25 @@ def test_get_model() -> None:
         model.measurement_aggregate_document.measurement_identifier = ""
 
     assert model == get_model()
+
+
+@pytest.mark.parametrize("output_file", output_files)
```
Comment (nathan-stender) [inline tests/parsers/perkin_elmer_envision/elmer_envision_parser_test.py:43]: Let's remove the parameterization on this test, we only need one instance of this.

File: tests/parsers/perkin_elmer_envision/elmer_envision_parser_test.py [hunk 3]
```diff
@@ -37,3 +38,25 @@ def test_get_model() -> None:
...
+def test_parse_missing_file(output_file: str) -> None:
+    test_filepath = f"tests/parsers/perkin_elmer_envision/testdata/{output_file}.tsv"
+    with pytest.raises(FileNotFoundError):
+        from_file(test_filepath, VENDOR_TYPE)
+
+
+@pytest.mark.parametrize("output_file", output_files)
```
Comment (nathan-stender) [inline tests/parsers/perkin_elmer_envision/elmer_envision_parser_test.py:50]: Same here, no need to parameterize.

**Review by nathan-stender — CHANGES_REQUESTED** [review]
Thank you for adding this, one more request, and then I think this will be all good to merge!

**Comment by mdlu** [comment]
also be sure to merge/rebase on main, looks like there are some conflicts to be resolved first 🙏

**Comment by gvwilson** [comment]
happy to take this one on

File: tests/parsers/perkin_elmer_envision/elmer_envision_parser_test.py (hunk 3 above)
Comment (mwkyuen) [reply tests/parsers/perkin_elmer_envision/elmer_envision_parser_test.py]: Resolved in [5833c79](https://github.com/Benchling-Open-Source/allotropy/pull/72/commits/5833c79bdfb6cabff12553304bb83d5ba3dfdf67)

File: tests/parsers/perkin_elmer_envision/elmer_envision_parser_test.py (hunk 2 above)
Comment (mwkyuen) [reply tests/parsers/perkin_elmer_envision/elmer_envision_parser_test.py]: Resolved in [5833c79](https://github.com/Benchling-Open-Source/allotropy/pull/72/commits/5833c79bdfb6cabff12553304bb83d5ba3dfdf67)

**Comment by gvwilson** [comment]
@mdlu we think this one is ready for a look, but the CI job seems to be blocked waiting on... I'm not sure what?

**Comment by mdlu** [comment]
@gvwilson it looks like some CI checks need to be manually triggered on our end, just went ahead and did that -- looks like some linting issues? I'll take a look again once resolved

**Pull request closed without merging**